IFComp/lib/IFComp/Form/Entry.pm

```diff
@@ -28,6 +28,20 @@ has_field 'blurb' => (
     type => 'TextArea',
 );
 
+has_field 'playtime' => (
+    type => 'Select',
+    label => 'Estimated play time',
+    empty_select => '',
+    options => [[
```

Comment:
Does it work if you update HTML::FormHandler?

(If so, yes, I should up the minimum version in the cpanfile...)